### PR TITLE
Update results paragraph

### PR DESCRIPTION
**Pull request by zietzm**
Addresses #37

### CODE REVIEW

File: content/02.body.md [hunk 1]
```diff
@@ -230,10 +230,12 @@ Only slightly better was its performance in predicting the separate TF-TG networ
 We find superior performance in predicting the co-authorship relationships (AUROC 0.75), which was expected as the network being predicted shared roughly the same degree distribution as the network on which the edge prior was computed.
 The results of the third prediction task show that a difference in degree distribution between the network on which features are computed and the network to be predicted can make prediction significantly more challenging.
 
-Since the edge prior is based only on degree, it is unsurprising that it exhibits weak performance in predicting a network with a different degree distribution.
-We have considered the edge prior as a baseline edge predictor, whose performance indicates the utility of degree for a specific prediction task.
+The edge prior can be considered a baseline edge predictor that excellently captures degree's contribution to the probability of an edge existing.
```
Comment (cgreene) [inline content/02.body.md]:
```suggestion
The edge prior can be considered a baseline edge predictor that accurately captures degree's contribution to the probability of an edge existing.
```

File: content/02.body.md [hunk 2]
```diff
@@ -230,10 +230,12 @@ Only slightly better was its performance in predicting the separate TF-TG networ
...
 The edge prior's low performance in the third task indicates that degree is less helpful for edge prediction tasks in which training and testing networks do not share their degree distributions.
-Moreover, we believe such between-distribution prediction may be a relatively common task, with examples given by the networks in Figure {@fig:degree-bias}.
+Many biomedical prediction tasks can be framed as edge prediction tasks between different degree distributions.
+In drug repurposing, for example, existing compound-disease treatment relationships are not randomly sampled from all true treatment relationships.
```
Comment (cgreene) [inline content/02.body.md]:
```suggestion
In drug repurposing, for example, existing compound-disease treatment relationships are unlikely to be randomly sampled from all true treatment relationships.
```

File: content/02.body.md [hunk 3]
```diff
@@ -230,10 +230,12 @@ Only slightly better was its performance in predicting the separate TF-TG networ
...
+However, all treatment relationships between existing compounds and diseases are desirable outputs in prediction.
+Framed as edge prediction in this way, drug repurposing attempts to predict between different degree distributions.
```
Comment (cgreene) [inline content/02.body.md]:
```suggestion
Understanding the extent to which degree influences accuracy metrics can reveal the extent to which predictions from one or more networks are due to underlying biological properties, as opposed to network degree distributions, which may be consistent and generalize to the true network which has a different distribution.
```
I do not really like this - but I think it's the sentiment that's missing.

**Review by cgreene — CHANGES_REQUESTED** [review]
I have a couple changes. I think the final sentence sentiment should somehow be here. However, I don't like the framing I suggested. It was all I could come up with before a meeting that I need to run to.

File: content/02.body.md (hunk 3 above)
Comment (zietzm) [reply content/02.body.md]: I agree that this sentiment is a good way to conclude this section and motivate the section that follows. How about the following:

```suggestion
Edge predictions can be based on both underlying biological properties and network degree distributions.
However, predictions based on biological properties may be more consistent and generalizable than those based on degree.
Degree's influence on edge prediction accuracy measures can reveal the relative contributions of these two factors.
```

Comment (cgreene) [reply content/02.body.md]: I agree with the change above 👍

**Review by cgreene — APPROVED** [review]

**Comment by zietzm** [comment]
Closed #37

**Pull request merged**